tests/test_restful_api.py

```diff
@@ -20,76 +20,14 @@
 
 """claimstore.modules.claims.restful test suite."""
 
-import json
-import os
+from claimstore.modules.claims.fixtures.claim import CLAIM_CDS1_FN, \
+    CLAIM_INSPIRE1_FN, CLAIM_INSPIRE2_FN, create_claim, load_claim
+from claimstore.modules.claims.fixtures.claimant import CLAIMANT_CDS_FN, \
+    CLAIMANT_INSPIRE_FN, create_claimant, load_claimant
+from claimstore.modules.claims.fixtures.predicate import create_all_predicates
+from webtest import TestApp  # isort: skip
```

Comment:
`s/isort: skip/isort:skip/`  

BTW I still get warnings locally, the best is perhaps to use the following `test_restful_api.py` module docstring:

```python
"""claimstore.modules.claims.restful test suite.

isort:skip_file
"""
```

This will switch off isort checking for the whole file.